releasenotes/notes/repro-v05-issues-cabf288f1f6cec59.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
```

Comment:
does this need a release note?

Reply:
Honestly no. I added one so I could backport this change to 1.20. I will remove it.